.github/workflows/cross-build.yaml

```diff
@@ -22,6 +22,8 @@ jobs:
             toolchain: ${{ matrix.rust }}
             target: ${{ matrix.target }}
             override: true
+      - name: Install arm64 libfdt
+        run: wget http://ftp.us.debian.org/debian/pool/main/d/device-tree-compiler/libfdt-dev_1.6.0-1_arm64.deb && dpkg-deb -xv libfdt-dev_1.6.0-1_arm64.deb ./tlibfdtdev && sudo mkdir /tmmmp && mkdir target && mkdir target/debug && mkdir target/debug/deps && sudo cp ./tlibfdtdev/usr/lib/aarch64-linux-gnu/libfdt.a target/debug/deps/libfdt.a && echo "libfdt installed"
```

Comment:
This could break if that particular version is removed from Debian. Can you not build it from source?

Reply:
This is a horrible workaround indeed... In fact this is the main reason of me creating #1263. I am thinking if we can apply an arm64 machine from packet.net and test the build of cloud-hypervisor on real arm64 machines instead of cross-building it on x86_64 machines.

Reply:
One of the benefits of cross-building is that we can results nice and quickly as part of the GitHub CI.

@MrXinWang If you can provide an ARM64 machine we can integrate in into our Jenkins CI via SSH which is something that will be necessary for running the integration tests too.

Reply:
@rbradford 

> One of the benefits of cross-building is that we can results nice and quickly as part of the GitHub CI.

I think we can also assign this arm64 machine to the github action?

> If you can provide an ARM64 machine we can integrate in into our Jenkins CI via SSH which is something that will be necessary for running the integration tests too.

Talked to my manager about this yesterday and I think we can, it will be similar thing in kata-container community.

Reply:
> @rbradford
> 
> > One of the benefits of cross-building is that we can results nice and quickly as part of the GitHub CI.
> 
> I think we can also assign this arm64 machine (or we can even use our own development machine) to the github action?

I don't know about that. If we have a fast and reliable full build CI then we can remove the GitHub action in the future.

> 
> > If you can provide an ARM64 machine we can integrate in into our Jenkins CI via SSH which is something that will be necessary for running the integration tests too.
> 
> Talked to my manager about this yesterday and I think we can (I mean applying from packet), and it will be similar thing in kata-container community. I think I will try to apply one next week (5pm local time here and it is getting hard to find people...).

Okay sounds good, thanks.

Reply:
@rbradford About the github action, see [here](https://help.github.com/en/actions/hosting-your-own-runners) about the self-host runners.